Fixes nits for #5758

arjansingh opened the pull request:
## Summary

1. Fixes nits for #5758
2. Adds some git ignores

┆Issue is synchronized with this [Notion page](https://www.notion.so/PR-5763-Fixes-nits-for-5758-2796d73d36508192b3e6feecfcacf38b) by [Unito](https://www.unito.io)

Commit:
[chore] ignore local Claude files

Commit:
[fix] simplify fix logic for error state

webfiltered reviewed (changes requested):
Humbly requesting you take my stuff:
- #5764

webfiltered committed:
Remove frankly unreasonable amount of bang-bang (#5764)

## Summary

Removes a frankly unreasonable amount of bang-bang.

## Changes

- Removes many bang-bang
- Adds one bang-bang to rule them all

## Review Focus

Focus on the bang-bang.

## Screenshots (if applicable)

<img width="20" height="19" alt="image"
src="https://github.com/user-attachments/assets/a33dad3c-351b-49ea-a6fd-d4cf102b0ff9"
/>

┆Issue is synchronized with this [Notion
page](https://www.notion.so/PR-5764-Remove-frankly-unreasonable-amount-of-bang-bang-2796d73d365081dc9fafcafd73ba8ff4)
by [Unito](https://www.unito.io)
